devel/libfmt/Portfile

```diff
@@ -17,11 +18,12 @@ long_description    \
     fmt (formerly cppformat) is an open-source formatting library. \
     It can be used as a safe alternative to printf or as a fast alternative to C++ IOStreams.
 
-checksums           rmd160  7ab5001c8144d8d3d25b1726295e1447d065d694 \
-                    sha256  7b40e266c16cbcc16a9d8743713d012be3188872c4bce5dde3556cdf3b5846d2 \
-                    size 662560
+checksums           rmd160  f93a5790bd795411ec6956a3470693cc3ad1e5e0 \
+                    sha256  9a6bb7dad07ad385d52597593f3cbc0d7180bfae87f2f2ede00b9ca6b801f48f \
+                    size 708513
 
 compiler.cxx_standard   2011
+compiler.blacklist-append {clang < 800}
```

Comment:
When blacklisting compilers like this, it's a good idea to add a comment explaining why it was necessary. For example, the comment could show the error message that earlier compilers generated, and/or a link to an upstream bug report about the problem.

Reply:
Got it - will keep that in mind!